hack/containerd.toml

```diff
@@ -1,12 +1,7 @@
 version = 2
 
-disabled_plugins = ["io.containerd.snapshotter.v1.aufs", "io.containerd.v1.zfs", "io.containerd.snapshotter.v1.zfs", "io.containerd.v1.devmapper", "io.containerd.snapshotter.v1.devmapper", "io.containerd.snapshotter.v1.btrfs"]
```

Comment:
in fact I moved `hack/containerd.toml` to `hack/cri-containerd.toml`, and created new `hack/containerd.toml`